packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/SpanDetail/KeyValuesTable.tsx

```diff
@@ -23,6 +23,8 @@ import { KeyValuePair, Link } from '../../../../types/trace';
 
 import './KeyValuesTable.css';
 
+const jsonObjectOrArrayStartRegex = new RegExp('^[{[]+');
```

Comment:
Why not use a regular expression literal? `/^(\{|\[)/`

Reply:
Makes sense, will update it later today(seems like the project can not be built on windows...) and push the changes. Thanks